db/migrate/20200511043818_add_more_flair_columns_to_group.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class AddMoreFlairColumnsToGroup < ActiveRecord::Migration[6.0]
+  def change
+    add_column :groups, :flair_icon, :string
+    add_reference :groups, :flair_image, foreign_key: { to_table: :uploads }
```

Comment:
`add_reference` is discouraged in our code base now. It'll be better to be more explicit about the commands here.

Reply:
Also I think `flair_upload_id` will be a better name since we're referencing an upload.